src/SignUpIn/SignIn.js

```diff
@@ -34,63 +63,97 @@ class SignIn extends Component {
+                                {/* TODO: Link straight to signup when login routing is ready */}
+                                {/* <Link to={'/signup'}>Create Account</Link> */}
+                                <div className="back" onClick={this.props.back}>
+                                    Back
+                                </div>
                             </div>
-                        </div>
                     </div>
+                
                 </form>
-            </div>
-        );
+            );
+        } else {
+            return (
+                <div className="signup-wrapper">
+                    
+                    <form onSubmit={this.signIn}>
+                        <div className="login-wrapper">
+                            <div className="login-input-wrapper">
+                                <input
+                                    name="email"
+                                    type="email"
+                                    id="email"
+                                    className="login-input form-component"
+                                    placeholder="Email"
+                                    required
+                                />
+                                <br />
+                                <input
+                                    name="password"
+                                    type="password"
+                                    id="password"
+                                    className="login-input form-component"
+                                    placeholder="Password"
+                                    required
+                                />
+                                <br />
+                            </div>
+                            {this.props.signInDenied && (
+                                <p className="sign-in-error">
+                                    Log in denied. Account is not verified or
+                                    activated.
+                                </p>
+                            )}
+                            {this.state.error && (
+                                /* TODO: give better error msg */
+                                <p className="sign-in-error">
+                                    Unable to log in.
+                                </p>
+                            )}
+                            <div className="login-button-wrapper">
+                                <button type="submit" className="login-button">
+                                    login
+                                </button>
+                            </div>
+                            <div className="forgot">
+                                {/* TODO: Add functionality to reset username and password */}
+                                {/*<p className="forgot">forgot password?</p> */}
+                                <button onClick={this.setForgot}>Forgot password?</button>
```

Comment:
I would try adding some CSS styling to this button to make it look like a smaller version of the "login" button right above. Check out the login-button-wrapper and login-button CSS classes and try to mess around with those. I know CSS isn't your biggest strength though so let me know if you're having trouble :)